docs/recipes/custom-transformer.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Custom transformer
+---
+
+## Use case
+
+Sometimes the default response serializer isn't what you need and you're looking for a way to overwrite the default serialization for a response.
```

Comment:
It would be worth mentioning what the "default response serializer" is. 

We can either briefly mention that by default MSW serializes `application/json` content type responses and this recipe is to opt-out from that behavior, or we can describe this in the "Response resolver" section of the docs and reference that section on this recipe's page.

Reply:
I did not know about this, I thought you always had to use `ctx.json` or `ctx.xml` and I didn't want to go into details in this recipe. I'll also link to the response resolver (I also didn't know this page existed), because it can be helpful.

Thanks for your review, I'll address the changes later today or tomorrow.